std-bits/table/src/main/java/org/enso/table/data/column/operation/map/MapOperationProblemBuilder.java

```diff
@@ -5,44 +5,36 @@
 import org.enso.table.data.table.problems.ArithmeticOverflow;
 import org.enso.table.data.table.problems.FloatingPointGrouping;
 import org.enso.table.data.table.problems.IllegalArgumentError;
-import org.enso.table.problems.AggregatedProblems;
+import org.enso.table.problems.ColumnAggregatedProblemAggregator;
+import org.enso.table.problems.ProblemAggregator;
 
 /**
  * This class is used to aggregate problems occurring during map operations performed on a storage.
  *
  * <p>A single instance of this builder should not be re-used for different map operations. It may
  * only be used with a single operation.
  */
-public class MapOperationProblemBuilder {
+public class MapOperationProblemBuilder extends ColumnAggregatedProblemAggregator {
```

Comment:
Maybe rename this to MapOperationProblemAggregator? At first I was confused and thought that aggregators and builders might be different things.

Reply:
Yes, I think you are right - the name is a leftover from the old approach and it can be confusing.  I will indeed rename it.